eledmac.dtx

```diff
@@ -10838,7 +10837,7 @@
 \fi
 %    \end{macrocode}
 % \subsubsection{Options for familiar notes}
-% Before generating the options commands for familiar notes, we check the \verb+nofamiliar+ option.
+% Before generating the optional commands for familiar notes, we check the \verb+nofamiliar+ option.
```

Comment:
quelque soit la solution retenue, il faudrait appliquer aussi à la ligne 10866 ("Before generating the options commands for endnotes")